047-TrafficControlWithDapr/Coach/README.md

```diff
@@ -1,26 +1,146 @@
 - Challenge 7: **[Dapr Secrets Management](Solution-07.md)**
-   - Add the Dapr secrets management building block.
+  - Add the Dapr secrets management building block.
 - Challenge 8: **[Dapr-enabled Services running in Azure Kubernetes Service (AKS)](Solution-08.md)**
-   - Deploy the Dapr-enabled services you have written locally to an Azure Kubernetes Service (AKS) cluster.
+  - Deploy the Dapr-enabled services you have written locally to an Azure Kubernetes Service (AKS) cluster.
+
+## Coach Prerequisites
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the `/Student/Resources` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.
+
+## Azure Requirements
+
+This hack requires students to have access to an Azure subscription where they can create and consume Azure resources. These Azure requirements should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students.
+
+To start, you'll need access to an Azure Subscription & Resource Group.
+
+You will need the following subcription [resource providers](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types) registered.
+
+- Microsoft.Cache
+- Microsoft.ContainerService
+- Microsoft.ContainerRegistry
+- Microsoft.Devices
+- Microsoft.EventHub
+- Microsoft.Insights
+- Microsoft.KeyVault
+- Microsoft.Logic
+- Microsoft.OperationalInsights
+- Microsoft.ServiceBus
+- Microsoft.Storage
+- Microsoft.Web
+
+- If you already have an Azure account, make sure you have at least [Contributor access instructions](https://docs.microsoft.com/azure/role-based-access-control/check-access) for the resource group in which you'll provision Azure resources.
+
+_Your IT organization may provide you access to an Azure resource group, but not the entire subscription. If that's the case, take note of that resource group name and make sure you have `Contributor` access to it, using the instructions mentioned above._
```

Comment:
I'm going to approve this for now.  However, this is written as-if the student is reading it.  This section should be targeted at the Coach and the customer stakeholder they will need to communicate these requirements too.  We should probably word it a little better.  

Instead of "You will need...", it should be "students will need..." etc.   This should change throughout this whole section above and below this comment.

Reply:
no problem, I can change this in either this PR or the next one